test/test_compile_benchmark_util.py

```diff
@@ -8,8 +8,6 @@
 
 
 try:
-    import tabulate  # noqa: F401  # type: ignore[import]
-
```

Comment:
Isn't it trying to test if this can be imported?

Reply:
Not. bench_all is imported in line 15 and its file has imported tabulate.